source/jormungandr/jormungandr/scenarios/new_default.py

```diff
@@ -173,6 +173,12 @@ def create_pb_request(requested_type, request, dep_mode, arr_mode):
     if request["free_radius_to"]:
         req.journeys.free_radius_to = request["free_radius_to"]
 
+    if request["min_nb_journeys"]:
```

Comment:
Doesn't this parameter clash with one that has the same name in https://github.com/CanalTP/navitia/pull/2413 ?

Reply:
What do you min by that? This isn't a new parameter, it exist since a few years

Reply:
There is no collision with min_nb_journeys parameters as Alex said, it exist since a few years

Reply:
:white_check_mark: